stores/moderationSubmissionsListStore.ts

```diff
@@ -0,0 +1,249 @@
+import { gql } from 'graphql-request'
```

Comment:
should rename this to just moderationSubmissionsStore. The list is the UI presentation part which should be abstracted from a store